qemu/target-i386/cpu.h

```diff
@@ -1137,6 +1137,10 @@ void cpu_x86_load_seg(CPUX86State *s, int seg_reg, int selector);
 void cpu_x86_fsave(CPUX86State *s, target_ulong ptr, int data32);
 void cpu_x86_frstor(CPUX86State *s, target_ulong ptr, int data32);
 
+/*  the binding language can not catch the exceptions.
+    check the arguments, return error instead of raise exceptions. */
+int _uc_check_cpu_x86_load_seg(CPUX86State *env, int seg_reg, int sel);
```

Comment:
how about removing the prefix _ in function name?

Reply:
Function name with underscore prefix means it is reserved, not public. Keep it is better.

Reply:
i know, but this is definitely not C convention ;-)

we never use this convention anywhere in Unicorn code, either.

Reply:
O.K. Remove the underscore.